src/gui/gui.ts

```diff
@@ -370,7 +380,7 @@ Do you want to install Bonjour?</source>
 
 %1
 
-This is a server fingerprint. You should compare this fingerprint to the one on your server&apos;s screen. If the two don&apos;t match exactly, then it&apos;s probably not the server you&apos;re expecting (it could be a malicious user).
+This is a server fingerprint. You should compare this fingerprint to the one on your server's screen. If the two don't match exactly, then it's probably not the server you're expecting (it could be a malicious user).
```

Comment:
This is mostly fine, but I'd rather we kept the `&apos;` here, as it's elsewhere in translations. Continuity, so to speak.